src/containers/pages/InterventionPlan/PlanDefinitionList/tests/__snapshots__/index.test.tsx.snap

```diff
@@ -1,7 +1,8 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
```

Comment:
@kelvin-muchiri unfortnately/fortunately you've run into one of those "poisoned chalice" components which still has a huge snapshot test :) - We need to make this snapshot smaller.

Reply:
@moshthepitt The snapshot is huge the fixtures are an array of plans which are then rendered as a table. Looks good to me. What do you think?

Reply:
@moshthepitt But yea..its possible to reduce it since we are not interested in testing the `ListView` but the data passed to it as prop

Reply:
@kelvin-muchiri we do not need the fixtures to appear in the Snapshot.  We can use [property matchers](https://jestjs.io/docs/en/snapshot-testing#property-matchers) for this.

- Just posting this for reference: [Effective snapshot testing](https://kentcdodds.com/blog/effective-snapshot-testing/)

Reply:
@moshthepitt Just looked at the component and remembered we do not need to about worry structure of the components since that should be handled by those components in their module tests. We should only be concerned about the props.